app/components/Canvas/Home/index.ts

```diff
@@ -16,6 +17,93 @@ import vertex from '../../../shaders/test-vertex.glsl'
 // @ts-ignore
 import fragment from '../../../shaders/test-fragment.glsl'
 
+const PROJECTS_MOCK = [
```

Comment:
Figure out a cleaner way to handle this code; possibly manage from the backend or extract out to a config json file instead.